### PR TITLE
840625 : Update the API Links in UG documentation in Maui PDF Viewer.

### PR DESCRIPTION
**Pull request by AntroJames**
Description : 
     Update the API Links in UG documentation in Maui PDFViewer.
                 1. Migration Document
                 2. Annotation Collection
                 3. Select and Deselect Annotations
                 4. Add, Remove and Edit Annotations
                 5. Lock and Unlock Annotations
                 6. Import and Export Annotations
                 7. Undo and Redo
                 8. Save a Document
                 9. Annotations

### CODE REVIEW

File: MAUI/PDF-Viewer/Import-Export-Annotations.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ documentation: ug
 
 # Import and Export Annotations in .NET MAUI PDF Viewer (SfPdfViewer)
 
-This section will go through the various functions available in the `SfPdfViewer` to import and export annotations in a PDF document.
+This section will go through the various functions available in the [Position](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.GestureEventArgs.html#Syncfusion_Maui_PdfViewer_GestureEventArgs_Position) to import and export annotations in a PDF document.
```
Comment (Deepak1799) [inline MAUI/PDF-Viewer/Import-Export-Annotations.md:12]: Please revisit this change

File: MAUI/PDF-Viewer/Lock-Unlock.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ You can lock an annotation to prevent it from being edited. The annotation that
 
 ## Lock all annotations in a document
 
-To lock all annotations in a document, set the `IsLocked` property of the `AnnotationSettings` to `true`. The following example explains how to lock all annotations in a document.
+To lock all annotations in a document, set the [IsLocked](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.BaseAnnotationSettings.html#properties) property of the [AnnotationSettings](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.AnnotationSettings.html) to `true`. The following example explains how to lock all annotations in a document.
```
Comment (Deepak1799) [inline MAUI/PDF-Viewer/Lock-Unlock.md:16]: Please revisit the **IsLocked** linking

File: MAUI/PDF-Viewer/Lock-Unlock.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ void LockCircleAnnotations()
 
 ## Lock the selected annotation
 
-To lock the selected annotation, access the selected annotation instance and set the `IsLocked` property of the annotation to `true`. The selected annotation instance may be obtained from the `AnnotationSelected` event. The following example explains how to lock the selected annotation in a PDF document.
+To lock the selected annotation, access the selected annotation instance and set the `IsLocked` property of the annotation to `true`. The selected annotation instance may be obtained from the [AnnotationSelected](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html#Syncfusion_Maui_PdfViewer_SfPdfViewer_AnnotationSelected) event. The following example explains how to lock the selected annotation in a PDF document.
```
Comment (Deepak1799) [inline MAUI/PDF-Viewer/Lock-Unlock.md:76]: image

It is missing in some of the places

File: MAUI/PDF-Viewer/Text-Markups.md [hunk 4]
```diff
@@ -145,7 +145,7 @@ void CustomizeDefaultHighlightSettings()
 
 ## Edit the selected text markup annotation
 
-You can edit the properties of the selected text markup annotation programmatically by accessing the selected annotation instance. The selected annotation instance may be obtained from the `AnnotationSelected` event. The following example shows how to edit some of the properties of the selected highlight annotation. Similarly, you can modify the other text markup annotations properties.
+You can edit the properties of the selected text markup annotation programmatically by accessing the selected annotation instance. The selected annotation instance may be obtained from the [AnnotationSettings](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html#Syncfusion_Maui_PdfViewer_SfPdfViewer_AnnotationSettings) event. The following example shows how to edit some of the properties of the selected highlight annotation. Similarly, you can modify the other text markup annotations properties.
```
Comment (Deepak1799) [inline MAUI/PDF-Viewer/Text-Markups.md:148]: Please revisit the linking

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
Requested changes

File: MAUI/PDF-Viewer/Lock-Unlock.md (hunk 2 above)
Comment (AntroJames) [reply MAUI/PDF-Viewer/Lock-Unlock.md]: Updated sir.

File: MAUI/PDF-Viewer/Import-Export-Annotations.md (hunk 1 above)
Comment (AntroJames) [reply MAUI/PDF-Viewer/Import-Export-Annotations.md]: Updated sir.

File: MAUI/PDF-Viewer/Lock-Unlock.md (hunk 3 above)
Comment (AntroJames) [reply MAUI/PDF-Viewer/Lock-Unlock.md:76]: Updated sir.

Comment (Deepak1799) [reply MAUI/PDF-Viewer/Lock-Unlock.md:76]: Still it is unresolved.

image

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
Requested changes

**Pull request merged**